src/engine/Settings.cs

```diff
@@ -133,6 +133,8 @@ private Settings()
     /// </summary>
     public SettingValue<bool> VolumeGUIMuted { get; set; } = new SettingValue<bool>(false);
 
+    public SettingValue<string> SelectedOutputDevice { get; set; } = new SettingValue<string>("Default");
```

Comment:
I think "Default" should be put in Constants to better explain that it is a built-in value that Godot recognizes.